test/Npgsql.Tests/SystemTransactionTests.cs

```diff
@@ -341,7 +489,7 @@ public void SchemaConnectionShouldntEnlist()
         }
 
         void AssertNoPreparedTransactions()
-            => Assert.That(GetNumberOfPreparedTransactions(), Is.EqualTo(0));
+            => Assert.That(GetNumberOfPreparedTransactions(), Is.EqualTo(0), "Prepared transactions found");
 
         int GetNumberOfPreparedTransactions()
```

Comment:
With commit 90f01ecc341fe11ab19464e4ad7cd795ae28249f, I have tried putting back that `AssertHasPreparedTransactions` in place for cases enlisting the test durable ressource, but it fails there too. Although I have seen that count raising in some circumstances when debugging...